src/nvim/extmark.c

```diff
@@ -568,8 +568,13 @@ void extmark_splice(buf_T *buf,
                     int new_row, colnr_T new_col, bcount_t new_byte,
                     ExtmarkOp undo)
 {
-  long offset = ml_find_line_or_offset(buf, start_row+1, NULL, true);
-  extmark_splice_impl(buf, start_row, start_col, offset+start_col,
+  long offset = ml_find_line_or_offset(buf, start_row + 1, NULL, true);
+
+  // TODO(vigoux): maybe filter a bit here...
+  if (offset < 0) {
```

Comment:
or dont do the extmark_splice_impl altogether ? 
ml_find_line_or_offset seems to return -1 for different usecases (for instance curline > buf->b_ml.ml_line_count) where I wonder if setting offset to 0 is the good recourse ?

Reply:
After some time looking at this it looks like the `ml_find_line_or_offset` call is actually valid, but fails for some reason...

Reply:
Better explanation on the comment I wrote above this very line.

Reply:
thanks for the explanation, the comment is helpful.